dev_tools/scripts/config

```diff
@@ -6,10 +6,26 @@
     - FixInclude: "hip/hip_runtime.h"
 - "(cuda|hip|omp|dpcpp)/test/factorization/par_ilu_kernels"
     - FixInclude: "core/factorization/par_ilu_kernels.hpp"
+- "(cuda|hip|omp|dpcpp)/test/factorization/par_ilut_kernels"
+    - FixInclude: "core/factorization/par_ilut_kernels.hpp"
+- "(cuda|hip|omp|dpcpp)/test/factorization/par_ict_kernels"
+    - FixInclude: "core/factorization/par_ict_kernels.hpp"
+- "cuda/factorization/par_ilut_select_common"
+    - FixInclude: "cuda/factorization/par_ilut_select_common.cuh"
+- "hip/factorization/par_ilut_select_common"
+    - FixInclude: "hip/factorization/par_ilut_select_common.hip.hpp"
+- "(cuda|hip|dpcpp)/factorization/par_ilut_"
+    - FixInclude: "core/factorization/par_ilut_kernels.hpp"
+- "(cuda|hip|dpcpp)/factorization/par_ict_"
+    - FixInclude: "core/factorization/par_ict_kernels.hpp"
 - "(cuda|hip|dpcpp)/preconditioner/jacobi_"
     - FixInclude: "core/preconditioner/jacobi_kernels.hpp"
 - "core/test/base/(extended_float|iterator_factory)"
     - RemoveTest: "true"
+- "core/test/base/allocator"
+    - FixInclude: "core/base/allocator.hpp"
+- "reference/test/base/utils"
+    - FixInclude: "core/base/utils.hpp"
```

Comment:
at some point, I will check whether adding a special comment as the main header.
```
#include "core/base/utils.hpp" // MAIN
```
it will use it as main header no matter how the config sets